src/components/calcite-card/calcite-card.scss

```diff
@@ -29,7 +28,11 @@
   @apply flex-auto flex flex-col;
 }
 
-:host([loading]) .calcite-card-container *:not(calcite-loader):not(.calcite-card-loader-container) {
+:host([loading]) .calcite-card-container {
+  @apply relative;
```

Comment:
@y0n4 can you comment on why we still need this container to be relative when loading?

Do you know why the relative messes up positioning after loading?

Reply:
I'm not sure for it to be just a Firefox browser issue. I'm thinking the relative positioning was affecting the following child's element with relative positioning even though it should be set to the boundaries of the dropdown position

Reply:
Interesting, I would expect the relative positioning to be a good thing.

Maybe this is something we can fix inside the dropdown component instead of having to fix it in other components?